### PR TITLE
Updates to get ecom service up.

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

**Comment by feanil** [comment]
@e0d

File: playbooks/edx-east/edx_service.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 - name: Build application artifacts
   hosts: all
   connection: local
-  gather_facts: True
+  gather_facts: False
```
Comment (maxrothman) [inline playbooks/edx-east/edx_service.yml:6]: Are you sure that we don't use gathered facts in any of our plays?

Comment (feanil) [inline playbooks/edx-east/edx_service.yml:6]: Yes, I've run these plays with this flag to set to false, when I was testing.

**Comment by maxrothman** [comment]
:+1:

**Pull request merged**